internal/pagination/pagination.go

```diff
@@ -56,19 +56,14 @@ type ListFilterFunc[T boundary.Resource] func(ctx context.Context, item T) (bool
 // If prevPageLastItem is empty, it returns a slice of T starting with the least recently updated.
 type ListItemsFunc[T boundary.Resource] func(ctx context.Context, prevPageLastItem T, limit int) ([]T, error)
 
-// ListRefreshItemsFunc returns a slice of T that have been updated since prevPageLastItem.
-// If prevPageLastItem is empty, it returns a slice of T that have been updated since the
-// item in the refresh token.
-type ListRefreshItemsFunc[T boundary.Resource] func(ctx context.Context, tok *refreshtoken.Token, prevPageLastItem T, limit int) ([]T, error)
-
 // EstimatedCountFunc is used to estimate the total number of items
 // available for the resource that is being listed.
 type EstimatedCountFunc func(ctx context.Context) (int, error)
 
 // ListDeletedIDsFunc is used to list the IDs of the resources deleted since
 // the given timestamp. It returns a slice of IDs and the timestamp of the
 // instant in which the slice was created.
-type ListDeletedIDsFunc func(ctx context.Context, since time.Time) ([]string, time.Time, error)
```

Comment:
We made the decision at this point to stop using the transaction timestamp in the return from the ListDeletedIDs callback. It was introduced _before_ we decided to add a database read timeout buffer to the queries. We could add this back, but we think that a `time.Now()` prior to making the query should provide sufficient accuracy (together with the buffer).

Reply:
OK, that is fine, we'll need to make that adjustment both to this function and the list functions since we use timestamps relative to that point as well.